CHANGELOG.md

```diff
@@ -18,7 +18,10 @@ This release has an [MSRV][] of 1.82.
 ### Added
 
 * Add `interpolate_unpremultiplied` and `gradient_unpremultiplied` for interpolating in unpremultiplied (straight) alpha space.
-  This allows using Color to implement web rendering features which specify such gradient rendering. ([#185][] by [@sagudev][])
+
+  While such interpolation will often give perceptually undesired results, this allows using Color to implement rendering features where such interpolation is specified or desired, like in the [HTML Canvas element][]. ([#185][] by [@sagudev][])
```

Comment:
```suggestion
  While such interpolation will often give perceptually undesired results, this allows using Color to implement rendering features where such interpolation is specified, like in the [HTML Canvas element][]. ([#185][] by [@sagudev][])
```

To make it even stronger. This is non-blocking; I'm not even convinced it's an improvement.

Reply:
> I'm not even convinced it's an improvement.

Fewer words!